lib/spack/spack/spec.py

```diff
@@ -4617,14 +4633,20 @@ def tree(self, **kwargs):
 
             if status_fn:
                 status = status_fn(node)
-                if node.installed_upstream:
+                if status == InstallStatus.external:
+                    out += clr.colorize("@g{[<]}  ", color=color)
```

Comment:
could we do `[e]` for external the `[<]` seems a bit misleading.

Reply:
I personally prefer `[x]` for eXternal

Reply:
We use `x` for eXplicit, I don't think it's a good idea to conflate the two.